source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/UtiliOS.mm

```diff
@@ -743,7 +743,12 @@ void buildIntermediateResultForText(ACRView *rootView, ACOHostConfig *hostConfig
                                                                   options:NSRegularExpressionSearch
                                                                     range:NSMakeRange(0, [parsedString length])];
     }
-
+    // Added manual replace of \\* when the text is used directly, html parsing is exluding this case successfully
+    if (!markDownParser->HasHtmlTags() && [parsedString containsString:@"\\*"]) {
```

Comment:
Is there a generic solution we can implement? This fix only accounts for * scenario.

For example, the same issue would occur for \\_SAMPLE TEXT\\_

Reply:
That's a good question, in the case the text is html, the parsing process cover these scenarios. This is when is no html text and the original string is used. I could create a generic version to replace the "\\" with the next character after the lines. My only doubt if applies for all scenarios and are no cases when this could mean something else.

Reply:
I have been doing some tests and, the rest of the scenarios are already covered, the "\\*" is the only scenario required because we don't have an alternative to put asterisks in a text block (single * is emphasis, ** is bolded and \* is not recognized by json parsing, even on web is not recognized). So, is ok to handle only this specific case. In the case you mentioned "\\_" is possible to display the "_" directly, not necessary any backslash

Reply:
Have you tested this payload?

`{
    "type": "AdaptiveCard",
    "version": "1.4",
    "body": [
        {
            "size": "Medium",
            "text": "Change Contacts \\_SAMPLE TEXT\\_ test",
            "type": "TextBlock"
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json"
}`

Desktop:
<img width="378" alt="image" src="https://github.com/microsoft/AdaptiveCards-Mobile/assets/42756862/09fe30e3-260d-4fb8-a97d-210dde1aa986">

iOS:
<img width="363" alt="image" src="https://github.com/microsoft/AdaptiveCards-Mobile/assets/42756862/03938d44-77f8-49b6-994b-328070bb89a7">

Reply:
Simplified solution to avoid visual diffs due to discussed change. After checking documentation, only 2 symbols ("*", "_") could take that behavior with double backslash.